hack/import-restrictions.yaml

```diff
@@ -50,6 +50,12 @@
   - k8s.io/kube-openapi
   - k8s.io/klog
 
+- baseImportPath: "./vendor/k8s.io/component-base/"
+  allowedImports:
+  - k8s.io/apimachinery
```

Comment:
Will `k8s.io/apimachinery` import packages from `k8s.io/component-base` in the future? I think the answer is yes. Correct me if I am wrong. To avoid import circle, we'd better add a TODO or create an issue to address this after the whole refactoring.

Reply:
No, component-base will always depend on apimachinery.